glean-core/examples/sample.rs

```diff
@@ -62,7 +62,7 @@ fn main() {
 
     let ping_maker = PingMaker::new();
     let ping = ping_maker
-        .collect_string(glean.storage(), glean.get_ping_by_name("core").unwrap())
+        .collect_string(&glean, glean.get_ping_by_name("core").unwrap())
```

Comment:
nit: it's totally unrelated, but can we rename `"core"` to `"baseline"` or something else?

Reply:
Taking that into #122